pennylane_forest/qpu.py

```diff
@@ -246,17 +221,19 @@ def expval(self, observable):
                         label, p00=self.readout_error[0], p11=self.readout_error[1]
                     )
 
+            prep_prog.wrap_in_numshots_loop(self.shots)
```

Comment:
@AlbertMitjans RE: https://github.com/PennyLaneAI/pennylane-forest/pull/107#discussion_r994449879

Where my confusion came from was that I wasn't sure how this code was working before. The `Program` wasn't being wrapped in a numshots loop and the number of shots wasn't being set on the Experiment either. Upon further investigation it turns out in `pyQuil` v2 an `Experiment` would default to 10k shots if it wasn't set on the `Program` or the `Experiment`.

Before, the code wasn't respecting the number of shots set by the user but it still passed all the tests because it defaulted to 10k shots. Since `Experiment`s no longer set shots in `pyQuil` v3 the program was only running a single time, causing the tests to fail due to inaccuracy and revealing the bug 🎊.

Given this, I removed the comment.